interface/VideoEncoderDefs.h

```diff
@@ -164,6 +169,7 @@ typedef struct VideoRateControlParams {
     int8_t diffQPIP;// P frame qp minus initQP
     int8_t diffQPIB;// B frame qp minus initQP
     uint32_t layerBitRate[32]; // specify each scalable layer bitrate
+    EncMiscParameterHRD encMiscParameterHRD;
```

Comment:
Do not put this in common, it's not normally used feature, normally, people do not need care about this